shadow-dom/focus-navigation/resources/focus-utils.js

```diff
@@ -1,17 +1,27 @@
 'use strict';
 
-function navigateFocusForward() {
-  // TAB = '\ue004'
-  return test_driver.send_keys(document.body, "\ue004");
+function waitForRender() {
+  return new Promise(resolve => requestAnimationFrame(() => requestAnimationFrame(resolve)));
+}
+
+async function navigateFocusForward() {
+  await waitForRender();
+  const kTab = '\uE004';
+  await new test_driver.send_keys(document.documentElement,kTab);
```

Comment:
@dizhang168 @josepharhar Is there a reason of why this gets changed to `document.documentElement` instead of `document.body`?

Reply:
Yeah, that looks good, it'd be great if you can make a patch! 

Using `document.documentElement` [doesn't work](https://bugzilla.mozilla.org/show_bug.cgi?id=1892334#c3) in current Gecko's setup.

Reply:
Great, thanks for raising this.
https://chromium-review.googlesource.com/c/chromium/src/+/5519292